apps/www/src/registry/examples/playground-demo.tsx

```diff
@@ -36,6 +37,7 @@ export default function PlaygroundDemo({
       plugins: [
         ...CopilotKit,
         ...EditorKit,
+        ...ExcalidrawKit,
```

Comment:
Could you add excalidraw-kit to `apps/www/src/registry/registry-kits.ts` and to `apps/www/src/registry/registry-examples.ts`. AI should do it well